.gitignore

```diff
@@ -1,2 +1,3 @@
 target
 Cargo.lock
+.*.swp
```

Comment:
What is this?

Reply:
Ah it was for vim swap files - was gitignoring it.